core/src/main/java/org/mule/execution/TransactionalErrorHandlingExecutionTemplate.java

```diff
@@ -80,18 +80,6 @@ public static TransactionalErrorHandlingExecutionTemplate createMainExecutionTem
     }
 
     /**
-     * Creates a TransactionalErrorHandlingExecutionTemplate to be used as first processing template in a flow using no particular exception listener.
-     * Exception listener configured in the flow within this ExecutionTemplate is executed will be used.
-     *
-     * @param muleContext MuleContext for this application
-     * @param transactionConfig Transaction configuration
-     */
-    public static TransactionalErrorHandlingExecutionTemplate createMainExecutionTemplate(MuleContext muleContext, TransactionConfig transactionConfig)
```

Comment:
Deprecate instead of remove (unless you check with PLG and he thinks it's ok to just remove)